README.md

```diff
@@ -1,28 +1,15 @@
 
-## Getting Started
-
-This repository includes files with plain SQL that can be used to recreate a database:
-
-- Use [schema.sql](./schema.sql) to create all tables.
-- Use [data.sql](./data.sql) to populate tables with sample data.
-- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
-
 <a name="readme-top"></a>
+<img src = 'https://user-images.githubusercontent.com/93039359/243916263-02370431-b4b5-474a-85bb-0f6fb15de502.png'>
 
```

Comment:
**[OPTIONAL]**

- [ ] 😄 I have suggested you make your own logo and give you my logo as an example 😄 but you are using my logo in your readme file but no worries you can use it but it would be better if you use your own logo 👍